packages/features/shell/navigation/Navigation.tsx

```diff
@@ -131,33 +142,36 @@ const platformNavigation: NavigationItemType[] = [
   },
 ];
 
-export const getDesktopNavigationItems = (isPlatformNavigation = false) => {
-  const navigationType = !isPlatformNavigation ? navigation : platformNavigation;
-  const moreSeparatorIndex = navigationType.findIndex((item) => item.name === MORE_SEPARATOR_NAME);
+const useNavigationItems = (isPlatformNavigation = false) => {
```

Comment:
It was exported but never used anywhere. So I removed `export` keyword and turned it into a hook (so that I can use `useOrgBranding()` hook inside)